src/plugins/gui/providers/banxaProvider.ts

```diff
@@ -759,215 +761,232 @@ export const banxaProvider: FiatProviderFactory = {
+                        const edgeTx = await showUi.send(sendParams)
+
+                        // At this point we'll call it success
+                        clearInterval(interval)
+                        interval = undefined
+
+                        await showUi.trackConversion('Sell_Success', {
+                          conversionValues: {
+                            conversionType: 'sell',
+                            destFiatCurrencyCode: fiatCurrencyCode,
+                            destFiatAmount: priceQuote.fiat_amount,
+                            sourceAmount: new CryptoAmount({
+                              currencyConfig: coreWallet.currencyConfig,
+                              currencyCode: displayCurrencyCode,
+                              exchangeAmount: coinAmount
+                            }),
+                            fiatProviderId: providerId,
+                            orderId: id
+                          }
+                        })
+
+                        // Below is an optional step
+                        const { txid } = edgeTx
+                        // Post the txid back to Banxa
+                        const bodyParams = {
+                          tx_hash: txid,
+                          source_address: receiveAddress.publicAddress,
+                          destination_address: publicAddress
                         }
-                        insideInterval = false
+                        await banxaFetch({
+                          method: 'POST',
+                          url,
+                          hmacUser,
+                          path: `api/orders/${id}/confirm`,
+                          apiKey,
+                          bodyParams
+                        }).catch(e => {
+                          console.error(String(e))
+                        })
+                      }
+                      insideInterval = false
+                    } catch (e: unknown) {
+                      if (
+                        e instanceof Error &&
+                        e.message === SendErrorBackPressed
+                      ) {
+                        await showUi.exitScene()
+                      } else if (
+                        e instanceof Error &&
+                        e.message === SendErrorNoTransaction
+                      ) {
+                        await showUi.exitScene()
+                        await showUi.showToast(
+                          lstrings.fiat_plugin_sell_failed_to_send_try_again
+                        )
+                      } else {
+                        await showUi.showError(e)
                       }
-                    }, 3000)
+                      insideInterval = false
+                    }
                   }
+                  interval ??= setInterval(() => {
+                    statusUpdateAsync().catch((error: unknown) => {
+                      showError(error)
+                    })
```

Comment:
Optional: Using `setInterval` with async tasks is a bit dangerous - if they are running slow, we can get overlapping work. I would prefer to see `makePeriodicTask`, which will await the previous task before stating the next one.